controller_manager:
  ros__parameters:
    joint_trajectory_controller:
    type: "joint_trajectory_controller/JointTrajectoryController"

joint_trajectory_controller:
  ros__parameters:
    joints:
      - Joint_1
      - Joint_2
      - Joint_3
      - Joint_4
      - Joint_5
      - Joint_6
      - Joint_7

    command_interfaces:
      - position

    state_interfaces:
      - position
      # - velocity

    action_monitor_rate: 20.0

    allow_partial_joints_goal: false
    open_loop_control: true
    constraints:
      stopped_velocity_tolerance: 0.01
      goal_time: 0.0
      Joint_1:
        trajectory: 0.05
        goal: 0.03
      Joint_2:
        trajectory: 0.05
        goal: 0.03
      Joint_3:
        trajectory: 0.05
        goal: 0.03
      Joint_4:
        trajectory: 0.05
        goal: 0.03
      Joint_5:
        trajectory: 0.05
        goal: 0.03
      Joint_6:
        trajectory: 0.05
        goal: 0.03 
      Joint_7:
        trajectory: 0.05
        goal: 0.03      